frontend/src/components/MapView/LeftPanel/LeftPanelTabs/TabPanel.tsx

```diff
@@ -22,7 +22,7 @@ const TabPanel = memo(({ children, value, index, ...other }: TabPanelProps) => {
       }}
       {...other}
     >
-      {children}
+      {index === value && children}
```

Comment:
I add this to nos render contents until tab is selected. Else it is rendered but not visible.